lopper_sanity.py

```diff
@@ -2468,6 +2506,7 @@ def usage():
     print('  -t, --tree          run lopper tree tests' )
     print('  -l, --lops          run lop tests' )
     print('  -o, --openamp       run openamp tests')
+    print('  -x, --xlnx          run xilinx-amd sanity tests')
```

Comment:
We should use a different / more generic name for this. If we start adding each user's sanity tests via a name, it will both the a maintenance issue and will be too specific.

Can we just call this "generate-domain"

Also, it needs to be run when --all is passed, which is currently not the case.